### ISSUE TITLE
Material Box inside card

### ISSUE THREAD

**Issue by braginteractive** [Dogfalo/materialize#568]
It looks like the lightbox is constrained inside the width of the container when used with a card. 

````
<div class="card">
    <div class="card-image">
             <img width="1200" height="800" src="http://imagepath" class="materialboxed" alt=alt text">
    </div>
     <span class="card-title">Hello world!</span>		
		<div class="card-content">
			<p>content of card</p>	
		</div> <!-- .card-content -->		
	</div>
````

**Comment by acburst** [comment]
Fixed in e39958fa2c133b650095163ecd3062f6f955ecfd

**Comment by RogerAntonio** [comment]
The same happens when the image is inside a container with class "side-nav"
